File: frontend/src/dtos/DetalleVenta/getById/DetalleVentaGetByIdResponseDto.ts
import LibroGetByIdResponseDto from "../../Libro/getById/LibroGetByIdResponseDto";
export default interface DetalleVentaGetByIdResponseDto
{
	detalleVentaId : number,
	cantidad : number
	subtotal : number

	libro : LibroGetByIdResponseDto;

}

File: backend/src/main/java/com/example/template/dtos/DetalleVentaGetByIdResponseDTO.java
package com.example.template.dtos;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import java.util.*;
import com.example.template.enums.*;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class DetalleVentaGetByIdResponseDTO {
private long DetalleVentaId;
private int Cantidad;
private float Subtotal;
private LibroGetByIdResponseDTO Libro;
}
